### PR TITLE
feat(SDK): add simulator

### PR DESCRIPTION
**Pull request by virror**
Add new Simulator SDK for using mouse instead of controllers.
Take 2!

### CODE REVIEW

File: Assets/VRTK/SDK/Simulator/SDK_ControllerSim.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        {
+            return selected;
+        }
+        set
+        {
+            selected = value;
+        }
+    }
+
+    private void Awake()
```
Comment (thestonefox) [inline Assets/VRTK/SDK/Simulator/SDK_ControllerSim.cs]: private methods should come after public and protected, so just move Awake and Update to the bottom of the file.

File: Assets/VRTK/SDK/Simulator/SDK_ControllerSim.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
+﻿using UnityEngine;
```
Comment (thestonefox) [inline Assets/VRTK/SDK/Simulator/SDK_ControllerSim.cs]: should this be in a namespace of VRTK?

File: Assets/VRTK/SDK/Simulator/SDK_InputSimulator.cs [hunk 3]
```diff
@@ -0,0 +1,227 @@
+﻿namespace VRTK
+{
+    using UnityEngine;
+
+    /// <summary>
+    /// Use the mouse and keyboard to move around both player and hands and interacting with objects
+    /// without the need of a hmd or VR controls.
+    /// </summary>
+    /// <remarks>
```
Comment (thestonefox) [inline Assets/VRTK/SDK/Simulator/SDK_InputSimulator.cs]: Don't include the remarks or example tags if they're empty

File: Assets/VRTK/SDK/SteamVR/SDK_SteamVRController.cs [hunk 4]
```diff
@@ -526,7 +526,6 @@ public override bool IsGripTouchedUpOnIndex(uint index)
             return IsButtonPressed(index, ButtonPressTypes.TouchUp, SteamVR_Controller.ButtonMask.Grip);
         }
```
Comment (thestonefox) [inline Assets/VRTK/SDK/SteamVR/SDK_SteamVRController.cs]: not sure why this is saying this file has been changed?

File: README.md [hunk 5]
```diff
@@ -63,7 +64,30 @@ online at [http://docs.vrtk.io](http://docs.vrtk.io).
...
+
+  * Drag the `VRTK_SimPlayer` prefab from the VRTK/Prefabs into the
+  scene.
+  * Select the GameObject with the `VRTK_SDKManager` script attached
+ to it.
+   * Select `Simulator` for each of the SDK Choices.
+   * Click the `Auto Populate Linked Objects` button to find the
+   relevant Linked Objects.
+  * Use the Left Alt to switch between mouse look and move a hand.
+  * Press Tab to switch between left7right hands.
```
Comment (thestonefox) [inline README.md]: left7right, i'm guessing should be left/right?

**Comment by virror** [comment]
This should now be fixed.

**Pull request merged**